sim/src/main/scala/midasexamples/PlusArgsModule.scala

```diff
@@ -0,0 +1,59 @@
+//See LICENSE for license details.
+
+package firesim.midasexamples
+
+import chisel3._
+import chisel3.util._
+import chisel3.util.Enum
+import freechips.rocketchip.config.{Config, Field, Parameters}
+
+import midas.widgets._
+
+import midas.widgets.ResetPulseBridge
+
+/** Defines a test group with the id of 0
+  */
+class PlusArgsModuleTestConfigGroup68Bit
+    extends Config((site, here, up) => { case PlusArgsTestNumber =>
+      0
+    })
+
+/** Defines a test group with the id of 1
+  */
+class PlusArgsModuleTestConfigGroup29Bit
+    extends Config((site, here, up) => { case PlusArgsTestNumber =>
+      1
+    })
+
+case object PlusArgsTestNumber extends Field[Int]
+
+class PlusArgsModuleIO(val params: PlusArgsBridgeParams) extends Bundle {
+  // Output value that PlusArgs bridge gives us
+  val gotPlusArgValue = Output(UInt((params.width).W))
+}
+
+/** A DUT to demonstrate usage of a PlusArgsBridge. Two test groups exist, which are referred to from
+  * [[TutorialSuite.scala]]
+  */
+class PlusArgsDUT(implicit val p: Parameters) extends Module {
+
+  def testGroup0(): PlusArgsBridgeParams = {
```

Comment:
In the future you could use an enum-like thing to capture more semantic information about each test in the type itself. 

e.g.:
```

sealed trait PlusArgTest { def params: PlusArgsBridgeParams}
case object WidePlusArgTest extends PlusArgTest { def params = PlusArgsBridgeParams(name = "plusar_v=%d", default = BigInt("276783146634859761135"), width = 68)} 
...
case object PlusArgsTestTypeKey extends Field[PlusArgTest]

```
etc...